ui/library/test/utils/scroll.test.ts

```diff
@@ -182,3 +175,20 @@ describe('scrollToPosition', () => {
     targetDiv.remove();
   });
 });
+
+describe('calculateTop', () => {
+  it('should calculate correct value', () => {
+    expect(calculateTop(1104, 0, 24, 1056, 282.156)).equal(797);
+  });
+});
```

Comment:
This isn't a very good test suite. It only tests a single happy path. If it were to break, it's unclear why. Since the numbers are passed in as literals into the function, it's not clear what they refer to without looking at the source function. And the description is obvious...any test for a function calculating something is likely calculating the correct value or it would fail, so you are describing something about testing, not the function.